ui/app/scripts/app/controllers/apps.js

```diff
@@ -81,7 +81,7 @@ define(['model/pageable'], function (Pageable) {
 
                                 // Define function for obtaining properties of a app
                                 item.getProperties = function() {
-                                    return appService.getAppInfo(this.type, this.name);
+                                    return appService.getAppInfo(this.type, this.name).$promise;
```

Comment:
While it definitely makes sense fix this to return Promise, I wonder what made this work fine so far? ;)